frontend/services/opengl_glfw/OpenGL_GLFW_Service.hpp

```diff
@@ -106,6 +113,45 @@ class OpenGL_GLFW_Service final : public AbstractFrontendService {
 
     void create_glfw_mouse_cursors();
     void update_glfw_mouse_cursors(const int cursor_id);
+#else
+        // init API, e.g. init GLFW with OpenGL and open window with certain decorations/hints
+        bool init(const Config& config) {return 1;}
+        bool init(void* configPtr) override {return 1;}
+        void close() override {}
+
+        void updateProvidedResources() override {}
+        void digestChangedRequestedResources() override {}
+        void setRequestedResources(std::vector<FrontendResource> resources) override {}
```

Comment:
The multiplication of functions for GL/Non-GL is painful to look at and probably hard to maintain. Might it be possible to wrap the defult implementation into some kind of preprocessor macro? Something like
```
#ifdef WITH_GL
    #define GL_STUB ;
#else 
    #define GL_STUB {}
#endif // WITH_GL

void maybe_gl_function() GL_STUB;
```
As is common for library symbols.